packages/app_ui/lib/src/theme/app_theme.dart

```diff
@@ -1,7 +1,12 @@
+import 'package:app_ui/src/colors/app_colors.dart';
 import 'package:flutter/material.dart';
 
 class AppTheme {
- static ColorScheme lightColorScheme = const ColorScheme.light();
- static ColorScheme darkColorScheme = const ColorScheme.dark();
- static const Color barrierColor = Colors.black54;
-}
\ No newline at end of file
+  static ThemeData get light {
+    return ThemeData(
+      fontFamily: 'Poppins',
```

Comment:
move it to the static const, e.g: before line 5.

`class AppTheme {`
`stati const _fontFamily = 'Poppins';`
`  static ThemeData get light {`
`    return ThemeData(`
`      fontFamily: _fontFamily,`